apps/etterlatte-saksbehandling-ui/client/src/components/behandling/felles/utils.ts

```diff
@@ -101,3 +101,7 @@ export const sisteBehandlingHendelse = (hendelser: IHendelse[]): IHendelse => {
 
   return hendelserSortert[hendelserSortert.length - 1]
 }
+
+export function hasValue<T>(value?: T | null): value is T {
+  return value !== undefined && value !== null
```

Comment:
God løsning hvis typesikkerhet er nødvendig, om ikke så vil `value != null` dekke både `null` og `undefined`

Reply:
Ja, det er kanskje en rimelig løsning også -- men jeg gikk bare for eksplisitt sjekk for begge tilfellene og så slipper man å huske de litt rare reglene med sammenligning i javascript :p